dynamiqs/utils/utils.py

```diff
@@ -378,6 +572,31 @@ def ket_to_bra(x: Tensor) -> Tensor:
 
     Returns:
         _(..., 1, n)_ Bra.
+
+    Examples:
+        >>> psi = dq.fock(3, 1)  # shape: (3, 1)
+        >>> psi
+        tensor([[0.+0.j],
+                [1.+0.j],
+                [0.+0.j]])
+        >>> dq.ket_to_bra(psi)  # shape: (1, 3)
+        tensor([[0.-0.j, 1.-0.j, 0.-0.j]])
```

Comment:
```suggestion
>>> psi = dq.fock(3, 1)       # shape: (3, 1)
>>> bra = dq.ket_to_bra(Psi)  # shape: (1, 3)
>>> bra == psi.mH
True
```
More straight to the point IMO